model/generate.go

```diff
@@ -270,9 +268,9 @@ func appendTasks(pairs TaskVariantPairs, bv parserBV, p *Project) TaskVariantPai
 // addGeneratedProjectToConfig takes a YML config and returns a new one with the GeneratedProject included.
 func (g *GeneratedProject) addGeneratedProjectToConfig(config string, cachedProject projectMaps) (string, error) {
```

Comment:
Shouldn't this function take a `parserProject` now instead of a config string?

Reply:
Is there a concern that the config string now has an outdated project? Should I put the parser project back into a string after this?

Reply:
> Is there a concern that the config string now has an outdated project? 

As long as we're certain that we'll never use the config string as long as a `parserProject` exists, I think that's fine. Is there anywhere we might be using the config string and not realize it? On the waterfall, for example?

> Should I put the parser project back into a string after this?

No, I think that would be expensive, and the main reason for doing this is make parsing projects cheaper.